src/averageAges.js

```diff
@@ -15,11 +15,11 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm'
+    && (!century || Math.ceil(person.died / 100) === century));
+
+  return men
+    .reduce((sum, person) => sum + person.died - person.born, 0) / men.length;
```

Comment:
It is better to create a variable before to make your code cleaner and more understandable